src/FlaUILibrary/flaui/module/mouse.py

```diff
@@ -76,20 +88,52 @@ def _click(element: Any):
         except NoClickablePointException:
             raise FlaUiError(FlaUiError.ElementNotClickable) from None
 
+    @staticmethod
+    def _click_hold(element: Any, timeout: int):
+        try:
+            FlaUI.Core.Input.Mouse.Position = element.GetClickablePoint()
+        except NoClickablePointException:
+            raise FlaUiError(FlaUiError.ElementNotClickable) from None
+        FlaUI.Core.Input.Mouse.Down()
+        time.sleep(float(timeout)/1000)
+        FlaUI.Core.Input.Mouse.Up()
+
     @staticmethod
     def _right_click(element: Any):
         try:
             return element.RightClick()
         except NoClickablePointException:
             raise FlaUiError(FlaUiError.ElementNotClickable) from None
 
+    @staticmethod
+    def _right_click_hold(element: Any, timeout: int):
+        try:
+            FlaUI.Core.Input.Mouse.Position = element.GetClickablePoint()
+        except NoClickablePointException:
+            raise FlaUiError(FlaUiError.ElementNotClickable) from None
+        FlaUI.Core.Input.Mouse.Down(FlaUI.Core.Input.MouseButton.Right)
+        time.sleep(float(timeout)/1000)
+        FlaUI.Core.Input.Mouse.Up(FlaUI.Core.Input.MouseButton.Right)
+
     @staticmethod
     def _double_click(element: Any):
         try:
             return element.DoubleClick()
         except NoClickablePointException:
             raise FlaUiError(FlaUiError.ElementNotClickable) from None
 
+    @staticmethod
+    def _double_click_hold(element: Any, timeout: int):
+        try:
+            FlaUI.Core.Input.Mouse.Position = element.GetClickablePoint()
+        except NoClickablePointException:
+            raise FlaUiError(FlaUiError.ElementNotClickable) from None
+        FlaUI.Core.Input.Mouse.Down()
+        FlaUI.Core.Input.Mouse.Up()
+        FlaUI.Core.Input.Mouse.Down()
+        time.sleep(float(timeout)/1000)

```

Comment:
Your timeout is already in ms so why you want to divide to 1000?

Your example is wrong as usage. It should be Clock Hold  <XPATH>  5000